p5-travel-app/src/client/js/restCountriesApi.js

```diff
@@ -1,12 +1,14 @@
+//https://restcountries.com/
 export async function getCountryInfo(countryCode) {
-  const endpoint = `https://restcountries.eu/rest/v2/alpha/${countryCode}`;
+  const endpoint = `https://restcountries.com/v3.1/alpha/${countryCode}`;
   try {
     const res = await fetch(endpoint);
     if (res.ok) {
       const data = await res.json();
+      console.log(data);
```

Comment:
remove

Reply:
done, clean it in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)